### DISCUSSION TITLE
bash::ubuntu error "Root process was alive less than 10 seconds"

### DISCUSSION THREAD

**Discussion by verajosemanuel** [microsoft/WSL#6496] [General]
# Environment
Microsoft Windows [Versión 10.0.19042.685]
Release 20.04
Linux version 4.19.128-microsoft-standard (oe-user@oe-host) (gcc version 8.2.0 (GCC)) #1 SMP Tue Jun 23 12:58:10 UTC 2020

# Steps to reproduce

1. configure bash::ubuntu setting a new "startup Task"

`%windir%\system32\bash.exe -cur_console:p`

2. Open new terminal: Ctrl + T
3. Choose the newly created bash::ubuntu and Error is shown

```
Command to be executed:
"C:\windows\system32\bash.exe"


ConEmuC: Root process was alive less than 10 sec, ExitCode=1.
Press Enter or Esc to close console...
```

#  Expected behavior

A new terminal for bash in ubuntu wsl should be opened

# Actual behavior

An error message is shown and I must force quit

**Comment by Biswa96** [comment]
ConEmu does not work with WSL2.
